src/vs/workbench/contrib/inlineChat/browser/inlineChatController.ts

```diff
@@ -320,7 +378,19 @@ export class InlineChatController implements IEditorContribution {
 				}
 			});
 
+			const enabledProvider = await this._enabledProvider;
+			if (!enabledProvider) {
+				this._dialogService.info(localize('create.failSlowExtHost', "Failed to start editor chat due to slow extension host"), localize('create.fail.detail', "Please consult the error log and try again later."));
```

Comment:
This is a message that we won't show the user but apart from that I doubt it is correct. The logic for `_enabledProvider` only allows for `undefined` when not having any provider.